marklogic-data-hub-central/ui/src/pages/Browse.tsx

```diff
@@ -160,23 +160,21 @@ const Browse: React.FC<Props> = ({ location }) => {
       }
       applySaveQuery(options);
     }
-    if(location.state && location.hasOwnProperty('zeroState') && !location.state['zeroState']){
-        setPageWithEntity(location.state['entity'],
-            location.state['pageNumber'],
-            location.state['start'],
-            location.state['searchFacets'],
-            location.state['query'])
-        location.state['tableView'] ? toggleTableView(true) : toggleTableView(false);
+
+    if (location.state && location.state.hasOwnProperty('zeroState') && !location.state['zeroState']) {
```

Comment:
i believe this is the fix for browser back button issue. Need a test for this.

Reply:
There are tests for that created before by Sanjeevni.

Reply:
That test doesnt verify the issue for HTTP 400 console error. The test passes I believe which indicates that it doesnt verify the issue in question. Other than console error we also get this:
<img width="1469" alt="image" src="https://user-images.githubusercontent.com/16124406/89360084-27c91600-d67c-11ea-92f3-ae523e2ad843.png">

 If this bug fixes these issues then we need a test for the same. Hope you have reproduced the issue and have steps to add a test for the same.

Reply:
The 400 error happened because of the incorrect api request caused by array in array in the searchOptions property. We do have explicit test for this case e.g. clicking back on details page and verifying searchOptions that failed before. 
  //Verify navigating back from detail view should persist search options
    detailPage.clickBackButton();
    browsePage.getSelectedEntity().should('contain', 'Person');
    browsePage.getClearFacetSearchSelection('Alice').should('exist');
    browsePage.getSearchText().should('have.value', 'Alice')
    browsePage.getTableView().should('have.css', 'background-color', 'rgb(68, 73, 156)')